internal/pkg/daemon/bpfrecorder/bpf/recorder.bpf.c

```diff
@@ -69,18 +76,38 @@ int sys_enter(struct trace_event_raw_sys_enter * args)
     // Update the command name if required
     char comm[MAX_COMM_LEN];
     bpf_get_current_comm(comm, sizeof(comm));
-    if (bpf_map_lookup_elem(&comms, &comm) == NULL) {
+    if (bpf_map_lookup_elem(&comms, &pid) == NULL) {
```

Comment:
Is this change by intention? Shouldn't be `comm` kept here?

Reply:
yes，This is the initial code written wrong，Because the key of comms map is pid.So we should use pid to query the comms map

Reply:
That's right. I submitted a fix just in case this doesn't get merged.

https://github.com/kubernetes-sigs/security-profiles-operator/pull/1311